### DISCUSSION TITLE
Welcome to aria Discussions!

### DISCUSSION THREAD

**Discussion by lef-fan** [lef-fan/aria#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by spnathan1** [comment]
can someone please explain to me like a noob how to install and use Aria? I have python and github desktop. I cloned the repo on github desktop but have no idea what to do next. Thanks

**Reply by adrew25** [reply]
Did you make a python3 environment to install the requirements.txt?

**Reply by spnathan1** [reply]
I'm not sure how to do that. I tried to do it just now after reading some google results, but am getting nowhere.

**Reply by lef-fan** [reply]
using pip:
`pip install -r requirements.txt`

then you need to follow installation instructions here:
[llama-cpp-python](https://github.com/abetlen/llama-cpp-python)

if you are using docker, you can run the client/server mode of Aria:
`docker buildx build --tag ghcr.io/lef-fan/aria-server:latest .`

**Reply by spnathan1** [reply]
I'm sorry, I'm really bad at this. I have pip installed, but when I open it, the window just disappears.

**Reply by lef-fan** [reply]
Hi.
Could you please be more specific, like what steps you have followed and what is currently the issue?
Thank you.
